### DISCUSSION TITLE
Image uploading problem

### DISCUSSION THREAD

**Discussion by dbelovol** [requarks/wiki#3194] [Help / Questions]
Hi!
I can't upload images. 
![Screenshot_20210310_004552](https://user-images.githubusercontent.com/49619457/110542537-37409e00-813a-11eb-98f3-86a526f1c954.png)
What could be the reason? May be because I use MySQL?
![Screenshot_20210310_005015](https://user-images.githubusercontent.com/49619457/110542841-9ef6e900-813a-11eb-98a1-0981134f6d75.png)

**Comment by dbelovol** [comment]
GraphQl request responds with 
```[{"errors":[{"message":"with recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFolders` where `id` = 0 union select `a`.`id`, `a`.`name`, `a`.`slug`, `a`.`parentId` from `assetFolders` as `a` inner join `ancestors` on `ancestors`.`parentId` = `a`.`id`) select * from `ancestors` - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFol' at line 1","locations":[{"line":3,"column":5}],"path":["assets","folders"],"extensions":{"code":"INTERNAL_SERVER_ERROR","exception":{"code":"ER_PARSE_ERROR","errno":1064,"sqlState":"42000","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFol' at line 1"}}}],"data":{"assets":{"folders":null,"__typename":"AssetQuery"}}},{"errors":[{"message":"with recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFolders` where `id` = 0 union select `a`.`id`, `a`.`name`, `a`.`slug`, `a`.`parentId` from `assetFolders` as `a` inner join `ancestors` on `ancestors`.`parentId` = `a`.`id`) select * from `ancestors` - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFol' at line 1","locations":[{"line":3,"column":5}],"path":["assets","list"],"extensions":{"code":"INTERNAL_SERVER_ERROR","exception":{"code":"ER_PARSE_ERROR","errno":1064,"sqlState":"42000","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFol' at line 1"}}}],"data":{"assets":{"list":null,"__typename":"AssetQuery"}}}]```

**Reply by dbelovol** [reply]
I'm sorry. [Here](https://docs.requarks.io/install/requirements/mysql5) is explanation
But I can't upload in the root too. Reports successful download, then aforementioned "unexpected error". Where /root should be located in local storage?

**Reply by NGPixel** [reply]
Upgrade to MySQL 8 or later
